tokenize_rt.py

```diff
@@ -82,29 +112,31 @@ def src_to_tokens(src):
     return tokens
 
 
-def tokens_to_src(tokens):
+def tokens_to_src(tokens):  # type: (Iterable[Token]) -> Text
     return ''.join(tok.src for tok in tokens)
 
 
 def reversed_enumerate(tokens):
+    # type: (Sequence[Token]) -> Iterator[Tuple[int, Token]]
```

Comment:
this type annotation should be `Generator[Tuple[int, Token], None, None]`

Reply:
I remember we talked about this but I don't remember what the reason is. Can you explain again? FWIW the docs mention that this syntax is OK:

https://github.com/python/mypy/blob/v0.710/docs/source/kinds_of_types.rst#Generators

Reply:
🏓

Reply:
Friendly ping :D

Reply:
oh, I explained this in the `pyupgrade` PR let me grab the copy from there:

> it is true that Generators are Iterators, but there are situations where an iterator cannot be used in place of a generator (for instance `@contextlib.contextmanager` or using `.send()`, etc.)